packages/modules/common/src/payments/citizen/response/index.js

```diff
@@ -113,8 +113,8 @@ export const SuccessfulPayment = (props) => {
   const amount = reciept_data?.paymentDetails?.[0]?.totalAmountPaid;
   const transactionDate = paymentData.transactionDate;
   const printCertificate = async () => {
-    const tenantId = Digit.ULBService.getCurrentTenantId();
-    const state = tenantId?.split(".")[0];
+    //const tenantId = Digit.ULBService.getCurrentTenantId();
+    const state = tenantId;
```

Comment:
Dont make these changes directly, 
verify both citizen and employee flow for payment in this case